### ISSUE TITLE
Task failed for domain_controller playbook

### ISSUE THREAD

**Issue by y0k4i-1337** [alebov/AD-lab#2]
I am facing an error in one of the tasks for domain_controller playbook. It might be related to Dsc module but I dont know how it works very well...
The error is the following:
<img width="1912" alt="task error" src="https://user-images.githubusercontent.com/7395296/119047812-c406a400-b994-11eb-985c-ed7ebd3d4bc9.png">
I tried to remove the Name parameter but then it shows another error and can't proceed. I would appreciate any suggestion on what to do to make it work.

**Comment by alebov** [comment]
Did you change some other setting for the machines?
Are you installing the dc on a machine running Windows Server 2019 or a previous version?

**Comment by y0k4i-1337** [comment]
I am using Windows Server 2019. Apparently they have been changing the xDnsServer Powershell module. I think the ansible task was not running because of that (just guessing).
Anyway, I managed to make it work on my machine by doing these changes: 31065375165b2eed5d0cd3160a0b312542994589 and d32d45e6fc5f1d81921acb02135a800d41f569a3.

I did that in order to use the new DnsServerDsc module. Since it is in prerelease version yet, I had to add some tasks based on this comment https://github.com/MicrosoftDocs/azure-docs/issues/29999#issuecomment-486728134.

Thanks for your response by the way :)
